util/test_timeline.ts

```diff
@@ -477,7 +477,7 @@ export const registerTestTimeline = (
 };
 
 class TestTimeline extends Timeline {
-  public override ui?: TestTimelineUI | undefined;
+  declare ui?: TestTimelineUI | undefined;
```

Comment:
This is because ts's class property is added before ES standard and have conflct.